src/lib/utils.test.js

```diff
@@ -84,6 +84,10 @@ describe("resolve", () => {
     expect(resolve("/groups?some=query", "/users?some=thing")).toEqual(
       "/groups?some=query"
     );
+    expect(resolve("users/.././users", "/")).toEqual("/users");
+    expect(resolve("some/.././users/../info/./address", "/users")).toEqual(
+      "/users/info/address"
```

Comment:
wouldn't this be `/info/address`?

```
/users
some -> /users/some
.. -> /users
. -> /users
.. -> /
info -> /info
. -> /info
address -> /info/address
```

Reply:
thanks to review, i will check it, and fixed the PR.

Reply:
thanks to review.
```js
// equal this:
/users/some/.././users/../info/./address
// delete /xx/..:
/users/./info/./address
// then:
/users/info/address
```
So I wonder if I missed it.？